LICENSE.txt

```diff
@@ -0,0 +1,214 @@
+LICENSE
+
+VMware vmw-guestinfo
+
+Copyright � 2017 VMware, Inc.  All rights reserved.

```

Comment:
That makes me realize that we should add proper copyright notice to all go files (instead of here)